cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/ClusterModel.java

```diff
@@ -1299,20 +1278,6 @@ public BrokerStats brokerStats(KafkaCruiseControlConfig config) {
     return utilization;
   }
 
-  /**
-   * Write to the given output stream.
-   *
-   * @param out Output stream.
-   */
-  public void writeTo(OutputStream out) throws IOException {
-    String cluster = String.format("<Cluster maxPartitionReplicationFactor=\"%d\">%n", _maxReplicationFactor);
-    out.write(cluster.getBytes(StandardCharsets.UTF_8));
-    for (Rack rack : _racksById.values()) {
-      rack.writeTo(out);
-    }
-    out.write("</Cluster>".getBytes(StandardCharsets.UTF_8));
-  }
-
```

Comment:
Why do we remove this method? I did a code search and I guess it is not related to this patch but because it is not used anywhere, right?

I noticed that `writeTo` method exists in `Replica`, `Broker`, `Load`, `Rack`, `Disk` and `Host`. Should we also keep it in `ClusterModel`?

Reply:
Ditto as above.